<launch>
  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" />
  <include file="$(find cordial_sprite)/launch/cordial_sprite.launch">
    <arg name="robot_name" value="DB1"/>
    <arg name="serial_port" value="/dev/ttyACM0"/>
    <arg name="use_tts" value="true"/>
    <arg name="phrase_file" value=""/>
    <!--<arg name="ivona_access_key" value="YOUR KEY HERE"/>-->
    <!--<arg name="ivona_secret_key" value="YOUR KEY HERE"/>-->
    <arg name="behavior_file" value="$(find cordial_experiment_example)/CoRDial_behaviors.json"/>
    <arg name="zeros" value="1300 1300 1500 1500 1441 1500"/>
  </include>
</launch>